priv/repo/migrations/20170424091310_soft_delete_user.exs

```diff
@@ -0,0 +1,9 @@
+defmodule CoursePlanner.Repo.Migrations.SoftDeleteUser do
+  use Ecto.Migration
+
+  def change do
+    alter table(:users) do
+      add :deleted_at, :utc_datetime
```

Comment:
I don't mind. Phoenix is going to support utc_datetime generators, but the standard will be naive_datetime.
Unless we're going to deal with timezones, we can set all to naive_datetime.